experimental/addons/elasticsearch-operator/README.md

```diff
@@ -0,0 +1,507 @@
+And the associated Pods:
+
+```shell
+$ kubectl get pod -n prod --selector='kibana.k8s.elastic.co/name=kibana'
+NAME                        READY   STATUS    RESTARTS   AGE
+kibana-kb-947c77847-kdbhf   1/1     Running   0          9m46s
+```
+
+**Access Kibana**
+
+A ClusterIP Service is automatically created for Kibana:
+
+```shell
+kubectl get service -n prod kibana-kb-http
+NAME             TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+kibana-kb-http   ClusterIP   10.109.2.144   <none>        5601/TCP   15m
+```
+
+```shell
+kubectl port-forward -n prod service/kibana-kb-http 5601
+```
+
+Get credentials: 
+
+```shell
+kubectl get secret -n prod elasticsearch-cluster-es-elastic-user -o=jsonpath='{.data.elastic}' | base64 --decode; echo
+```
+
+Login as the elastic user by visiting https://localhost:5601 in your browser.
+
+###  Elasticsearch agent
+
+**Deploy a Elasticsearch agent**
+
+Apply the following specification to deploy Elastic Agent with the System metrics integration to harvest CPU metrics from the Agent Pods. ECK automatically configures the secured connection to an Elasticsearch cluster named `elasticsearch-cluster` that you created previously.
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: elasticsearch-agent-sample
+spec:
+  components:
+    - type: elasticsearch-agent
+      name: elasticsearch-agent
+      properties:
+        version: 8.6.2
+        elasticsearchRefs:
+          - name: elasticsearch-cluster
+        daemonSet:
+          podTemplate:
+            spec:
+              securityContext:
+                runAsUser: 0
+        config:
+          inputs:
+            - name: system-1
+              revision: 1
+              type: system/metrics
+              use_output: default
+              meta:
+                package:
+                  name: system
+                  version: 0.9.1
+              data_stream:
+                namespace: default
+              streams:
+                - id: system/metrics-system.cpu
+                  data_stream:
+                    dataset: system.cpu
+                    type: metrics
+                  metricsets:
+                    - cpu
+                  cpu.metrics:
+                    - percentages
+                    - normalized_percentages
+                  period: 10s
+```
+
+Monitor the status of Elastic Agent.
+
+```shell
+$ kubectl get agent -n prod
+NAME                  HEALTH   AVAILABLE   EXPECTED   VERSION   AGE
+elasticsearch-agent   green    1           1          8.6.2     23m
+```
+
+List all the Pods that belong to a given Elastic Agent specification.
+
+```shell
+$ kubectl get pods -n prod --selector='agent.k8s.elastic.co/name=elasticsearch-agent'
+NAME                              READY   STATUS    RESTARTS   AGE
+elasticsearch-agent-agent-kfv9s   1/1     Running   0          21m
+```
+
+Access logs for one of the Pods.
+
+```shell
+$ kubectl logs -n prod -f elasticsearch-agent-agent-kfv9s
```

Comment:
we should able to see the logs in the elasticsearch portal instead of checking this logs